src/utils/components/HelpTextIcon/HelpTextIcon.scss

```
@@ -2,7 +2,8 @@
   cursor: pointer;
   color: var(--pf-t--global--icon--color--subtle);
 
-  transition: color var(--pf-t--global--motion--duration--sm)
+  transition:
+    color var(--pf-t--global--motion--duration--sm)
       var(--pf-t--global--motion--timing-function--default),
     transform var(--pf-t--global--motion--duration--sm)
       var(--pf-t--global--motion--timing-function--default);
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)